.github/workflows/build.yaml

```
@@ -22,7 +22,7 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v4
+    - uses: actions/checkout@v5
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)